sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RetryContext.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation;
+
+import java.util.List;
+
+public class RetryContext {
```

Comment:
Did manual testing , also planning to add mock tests to verify all the property of RetryContext in both connection mode and on various operations.

Reply:
Added the mock tests